src/open_inwoner/ssd/tests/test_utils.py

```diff
@@ -0,0 +1,77 @@
+from unittest import TestCase
+
+from parameterized import parameterized
+
+from ..xml import (
+    format_float_repr,
+    format_address,
+    format_date,
+    format_date_month_name,
+    format_name,
+    get_sign,
+    get_column,
+)
+
+
+class ParsingUtilsTest(TestCase):
+    @parameterized.expand([
```

Comment:
We should probably discus it if we want to add new infrastructure. Possibly there already experience to take advantage of. I guess this one is fine for now.

Reply:
Let's talk about it on Thursday; this stuff might not be needed if we decide to refactor.